plugins/formal-verification/src/org/jetbrains/kotlin/formver/conversion/ProgramConversionContext.kt

```diff
@@ -65,6 +69,9 @@ class ConvertedMethodSignature(val name: String, val params: List<ConvertedVar>,
         )
 }
 
+// Tracks the top-level information about the program.
+// Conversions for global entities like types should generally be
+// performed via this context to ensure they can be deduplicated.
```

Comment:
Same as the comment [here](https://kotlinlang.org/docs/kotlin-doc.html#block-tags).